src/main/java/org/isf/examination/gui/PatientExaminationEdit.java

```diff
@@ -1252,14 +1239,10 @@ public void stateChanged(ChangeEvent e) {
 	private ScaledJSlider getJSliderTemp() {
 		if (jSliderTemp == null) {
 			jSliderTemp = new ScaledJSlider(ExaminationParameters.TEMP_MIN, ExaminationParameters.TEMP_MAX, ExaminationParameters.TEMP_STEP, ExaminationParameters.TEMP_INIT);
-			jSliderTemp.addChangeListener(new ChangeListener() {
```

Comment:
removing this code means the imports for `ChangeEvent` and `ChangeListener` are no longer needed